File: test-fixtures/orm/sequelize/migration.js
// Sequelize migration
module.exports = {
  up: async (queryInterface, Sequelize) => {
    await queryInterface.createTable("Users", {
      id: { type: Sequelize.UUID, primaryKey: true, defaultValue: Sequelize.UUIDV4 },
      email: { type: Sequelize.STRING, allowNull: false, unique: true },
      password: { type: Sequelize.STRING, allowNull: false },
      ssn: { type: Sequelize.STRING },
      name: { type: Sequelize.STRING, allowNull: false },
      role: { type: Sequelize.ENUM("admin", "user"), defaultValue: "user" },
      createdAt: { type: Sequelize.DATE },
      updatedAt: { type: Sequelize.DATE },
    });
  },
  down: async (queryInterface) => {
    await queryInterface.dropTable("Users");
  },
};

File: drift v2/test-fixtures/orm/activerecord/migration.rb
# ActiveRecord migration
class CreateUsers < ActiveRecord::Migration[7.0]
  def change
    create_table :users do |t|
      t.string :email, null: false
      t.string :password, null: false
      t.string :ssn
      t.string :name, null: false
      t.integer :role, default: 0
      t.timestamps
    end

    add_index :users, :email, unique: true
  end
end
